README.md

```diff
@@ -30,89 +30,85 @@ Want to watch a screencast 🎥 about it? Check out https://symfonycasts.com/fou
-database-validate-mapping      Validate mapping in Zenstruck\Foundry\Tests\Fixtures\Entity
-database-drop-schema           Drop database schema
-composer                       Run composer command
-docker-start                   Build and run containers
-docker-stop                    Stop containers
-docker-purge                   Purge containers
-clear                          Start from a fresh install (use it for troubleshooting)
-```
+### Running the Test Suite
 
-Use double-dash to pass any PHPUnit options or arguments with `make`:
-```shell
-$ make test -- --stop-on-failure
-$ make test -- --filter FactoryTest
-# don't use "=" options value. ie: don't do this:
-$ make test -- --filter=FactoryTest
-```
+The test suite of this library needs one or more databases, then it comes with a docker compose configuration.
 
-Same syntax is available for composer:
-```shell
-$ make composer -- info symfony/*
-```
+> [!NOTE]
+> Docker and PHP installed locally (with `pgsql` & `mongodb` extensions) is required.
 
-#### Run tests in different environments
+You can start the containers and run the test suite:
 
-You can create a `.env` file to change the context in which tests will execute:
-```dotenv
-USE_ORM=1
-USE_ODM=1
-USE_DAMA_DOCTRINE_TEST_BUNDLE=1
-SYMFONY_REQUIRE=5.4.* # allowed values: 5.4.* | 6.0.* | 6.1.* | 6.2.*
-PHP_VERSION=8.0 # allowed values: 8.0 | 8.1 | 8.2
-PREFER_LOWEST=1 # force composer to request lowest dependencies
-```
+```bash
+# start the container
+$ docker compose up --build --detach
 
-### Change docker's ports
+# install dependencies
+$ composer update
 
-You can also add these variables to the `.env` file to change the ports used by docker:
-```dotenv
-PGSQL_PORT=5434
-MONGO_PORT=27018
+# run test suite with all available permutations
+$ composer test
+
+# run only one permutation
+$ vendor/bin/simple-phpunit
+
+# run test suite with dama/doctrine-test-bundle
+$ vendor/bin/simple-phpunit -c phpunit.dama.xml.dist
+
+# run test suite with postgreSQL instead of MySQL
+$ DATABASE_URL="postgresql://zenstruck:zenstruck@127.0.0.1:5433/zenstruck_foundry?serverVersion=15" vendor/bin/simple-phpunit
 ```
 
-### Execute commands in php container
+### Overriding the default configuration
+
+You can override default environment variables by creating a `.env.local` file, to easily enable permutations:
 
-You can execute any command into the php container using docker compose:
-```shell
-$ docker-compose exec php [your commmand] # or "docker compose" depending on your compose version
+```bash
+# .env.local
+DATABASE_URL="postgresql://zenstruck:zenstruck@127.0.0.1:5433/zenstruck_foundry?serverVersion=15"
+
+# run test suite with postgreSQL
+$ vendor/bin/simple-phpunit
 ```
 
-### Using xdebug with PhpStorm
+The `.env.local` file can also be used to override the port of the database containers,
+if it does not meet your local requirements. You'll also need to override docker compose configuration:
```

Comment:
I exactly had this problem: another project is using 3307, so I decided to put this in the docs. This is simple and could be useful

Reply:
Could we use a more obscure port as the default?

Reply:
I don't know... using just default port + 1 is kind of a convention... because it is possible to override everything, I think we can leave all these ports like this